rolling/release-el-build.yaml

```diff
@@ -0,0 +1,141 @@
+  emails:
+  - scott@openrobotics.org
+  maintainers: false
+package_blacklist:
+- async_web_server_cpp  # Not yet generated for RHEL
+- autoware_auto_msgs  # Not yet generated for RHEL
+- bno055  # python3-serial has no RPM package for RHEL
+- can_msgs  # Not yet generated for RHE
+- canopen_402  # Not yet generated for RHE
+- canopen_chain_node  # Not yet generated for RHE
+- canopen_master  # Not yet generated for RHE
+- canopen_motor_node  # Not yet generated for RHE
+- cartographer  # Requires Lua v5.2, system packages v5.1
+- cv_bridge  # Requires OpenCV v3, system packages v2
+- gazebo_dev  # Gazebo has no RPM package
+- grbl_msgs  # Not yet generated for RHEL
+- grbl_ros  # python3-serial has no RPM package for RHEL
+- image_tools  # Requires OpenCV v3, system packages v2
+- joy_teleop  # Not yet generated for RHEL
+- key_teleop  # Not yet generated for RHEL
+- mouse_teleop  # Not yet generated for RHEL
+- mrpt2  # ffmpeg has no RPM package in supported repositories
+- rc_common_msgs  # Not yet generated for RHEL
+- rc_dynamics_api  # Not yet generated for RHEL
+- rc_genicam_api  # Not yet generated for RHEL
+- rc_genicam_driver  # Not yet generated for RHEL
+- ros1_bridge  # ROS Noetic has no RPM packages for RHEL
+- ros2trace  # Not yet generated for RHEL
+- ros_canopen  # Not yet generated for RHE
+- sdformat_urdf  # Requires sdformat v9, system packages v2
+- serial_driver  # Not yet generated for RHEL
+- socketcan_bridge  # Not yet generated for RHE
+- socketcan_interface  # Not yet generated for RHE
+- teleop_tools  # Not yet generated for RHEL
+- teleop_tools_msgs  # Not yet generated for RHEL
+- test_osrf_testing_tools_cpp  # Doesn't install anything
+- tracetools  # Not yet generated for RHEL
+- tracetools_analysis  # jupyter-notebook has no RPM package for RHEL
+- tracetools_launch  # Not yet generated for RHEL
+- tracetools_read  # Not yet generated for RHEL
+- tracetools_test  # Not yet generated for RHEL
+- tracetools_trace  # Not yet generated for RHEL
+- ublox  # Not yet generated for RHEL
+- ublox_gps  # Not yet generated for RHEL
+- ublox_msgs  # Not yet generated for RHEL
+- ublox_serialization  # Not yet generated for RHEL
+- udp_driver  # Not yet generated for RHEL
+- udp_msgs  # Not yet generated for RHEL
+- webots_ros2  # Not yet generated for RHEL
+- webots_ros2_abb  # Not yet generated for RHEL
+- webots_ros2_core  # Not yet generated for RHEL
+- webots_ros2_demos  # Not yet generated for RHEL
+- webots_ros2_epuck  # Not yet generated for RHEL
+- webots_ros2_examples  # Not yet generated for RHEL
+- webots_ros2_importer  # Not yet generated for RHEL
+- webots_ros2_msgs  # Not yet generated for RHEL
+- webots_ros2_tiago  # Not yet generated for RHEL
+- webots_ros2_turtlebot  # Not yet generated for RHEL
+- webots_ros2_tutorials  # Not yet generated for RHEL
+- webots_ros2_universal_robot  # Not yet generated for RHEL
+- webots_ros2_ur_e_description  # Not yet generated for RHEL
+sync:
+  package_count: 262
```

Comment:
Does this package count take the blocked packages into account?

Reply:
It does not. We're still able to build far more than 262 packages, so it will probably work fine.